### PR TITLE
[common-artifacts] Remove nnpackage directory suffix

### PR DESCRIPTION
**Pull request by hseok-oh**
Remove nnpackage directory's OPT_FORMAT(.opt) suffix

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by mhs4670go** [comment]
@hseok-oh `opt` keyword is intended. Is there any problem?

**Comment by hseok-oh** [comment]
> Is there any problem?

No. I'm preparing re-enable nnpackage test (current nnpackage test job skip test because of invalid test list). And I found out directory have suffix. If you don't like this PR, I'll close this.

File: compiler/common-artifacts/CMakeLists.txt [hunk 1]
```diff
@@ -221,46 +221,46 @@ foreach(RECIPE IN ITEMS ${RECIPES})
...
+
     # Generate test directory
     set(TC_DIRECTORY "${NNPKG_PATH}/metadata/tc")
     add_custom_command(OUTPUT ${TC_DIRECTORY}
       COMMAND ${CMAKE_COMMAND} -E make_directory ${TC_DIRECTORY}
       DEPENDS ${NNPKG_PATH}
       COMMENT "Generate ${RECIPE} nnpackage test directory"
     )
-    
+
```
Comment (seanshpark) [inline compiler/common-artifacts/CMakeLists.txt:255]: @mhs4670go or @hseok-oh , if this PR is just closed, could you add another PR for these remove space changes?

Comment (mhs4670go) [reply compiler/common-artifacts/CMakeLists.txt:255]: I'll post a PR.
@hseok-oh Please close this PR:)

**Pull request closed without merging**